packages/components/organisms/f-checkout/test/accessibility/axe-accessibility.spec.js

```diff
@@ -65,4 +67,236 @@ describe('Accessibility tests', () => {
+    ])
+    .it('a11y - Authenticated - "%s" - should have a correct tab order in patch checkout error - "%s"', (serviceType, patchError) => {
+        // Arrange
+        checkout = new Checkout('organism', 'checkout-component');
+        checkout.withQuery('&knob-Service Type', serviceType)
+            .withQuery('&knob-Is User Logged In', true)
+            .withQuery('&knob-Patch Checkout Errors', patchError);
+
+        const pageUrl = buildUrl(checkout.componentType, checkout.componentName, checkout.path);
+
+        // Act
+        checkout.open(pageUrl);
+        checkout.waitForComponent();
+        checkout.goToPayment();
+
+        // Assert
+        const result = checkout.getCheckoutErrorTabOrder();
+        expect(result.actual).toEqual(result.expected);
+    });
+
+    forEach([
+        ['collection'],
+        ['delivery']
+    ])
+    .it('a11y - Authenticated - "%s" - should have a correct tab order in duplicate order error', serviceType => {
+        // Arrange
+        checkout = new Checkout('organism', 'checkout-component');
+        checkout.withQuery('&knob-Service Type', serviceType)
+            .withQuery('&knob-Is User Logged In', true)
+            .withQuery('&knob-Place Order Errors', 'duplicate');
+
+        const pageUrl = buildUrl(checkout.componentType, checkout.componentName, checkout.path);
+
+        // Act
+        checkout.open(pageUrl);
+        checkout.waitForComponent();
+        checkout.goToPayment();
+
+        // Assert
+        const result = checkout.getPlaceOrderErrorTabOrder();
+        expect(result.actual).toEqual(result.expected);
+    });
+
+    forEach([
+        ['collection', 'restaurant-not-taking-orders'],
+        ['collection', 'additional-items-required'],
+        ['collection', 'time-unavailable']
+    ])
+    .it('a11y - Guest - "%s" - should have a correct tab order in patch checkout error - "%s"', (serviceType, patchError) => {
+        // Arrange
+        checkout = new Checkout('organism', 'checkout-component');
+        checkout.withQuery('&knob-Service Type', serviceType)
+            .withQuery('&knob-Is User Logged In', false)
+            .withQuery('&knob-Patch Checkout Errors', patchError);
+
+        const pageUrl = buildUrl(checkout.componentType, checkout.componentName, checkout.path);
+        checkout.open(pageUrl);
+        checkout.waitForComponent();
+
+        // Act
+        checkout.setFieldValue('firstName', 'Jerry');
+        checkout.setFieldValue('lastName', 'Jazzman');
+        const addressInfo = {
+            emailAddress: 'jerry.jazzman@ronniescotts.co.uk',
+            mobileNumber: '+447111111111'
+        };
+        checkout.populateGuestCollectionCheckoutForm(addressInfo);
+        checkout.goToPayment();
+
+        // Assert
+        const result = checkout.getCheckoutErrorTabOrder();
+        expect(result.actual).toEqual(result.expected);
+    });
+
+    forEach([
+        ['delivery', 'restaurant-not-taking-orders'],
```

Comment:
Could these potentionally be added to the forEach above where we just have collection? (Line 125)

Reply:
They can't as delivery and collection have different form fields for the guest checkout state.
When you try to have them together and give it more fields to fill out than the page has, the test fails, citing that it can't find the relevant fields to populate. (e.g. address is not required in collection).